stock_by_warehouse/static/src/js/pop_over_locations.js

```diff
@@ -0,0 +1,41 @@
+odoo.define('stock_by_warehouse.pop_over_locations', function (require) {
+    'use strict';
+
+    var Widget = require('web.Widget');
+    var formats = require('web.field_utils');
+    var core = require('web.core');
+    var QWeb = core.qweb;
+
+    var PopOverLocations = Widget.extend({
```

Comment:
Can you name it without the PopOver part please.

Reply:
Done :smile: